bin/release

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+use Composer\Semver\VersionParser;
+use Tempest\Console\Console;
+use Tempest\Console\ConsoleApplication;
+use Tempest\Console\Exceptions\InterruptException;
+use function Tempest\get;
+use function Tempest\Support\arr;
+use function Tempest\Support\str;
+
+require_once getcwd() . '/vendor/autoload.php';
+
+function ensureAccess(string $remote, string $branch): void
+{
+    if (! empty(shell_exec('git status --porcelain 2>&1'))) {
+        throw new Exception('Repository must be in a clean state to release.');
+    }
+
+    if (! str_starts_with(shell_exec('git rev-parse --abbrev-ref --symbolic-full-name @{u}'), "$remote/$branch")) {
+        throw new Exception("You must be on the $remote/$branch branch to release.");
+    }
+}
+
+function getCurrentVersion(): string
+{
+    return exec('git describe --tags --abbrev=0');
+}
+
+function normalizeVersion(string $version): string
+{
+    return preg_replace('/^(\d+\.\d+\.\d+)\.0(-|$|\+)/', '$1$2', (new VersionParser())->normalize($version));
+}
+
+function suggestNextVersions(string $current): array
+{
+    $version = normalizeVersion($current);
+
+    if (! preg_match('/^(\d+)\.(\d+)\.(\d+)(?:-(?:alpha|beta)\.?(\d+))?$/', $version, $matches)) {
+        throw new InvalidArgumentException('Version must be in format X.Y.Z or X.Y.Z-alpha.N or X.Y.Z-beta.N');
+    }
+
+    $isStable = ! isset($matches[4]);
+    $major = (int) $matches[1];
+    $minor = (int) $matches[2];
+    $patch = (int) $matches[3];
+
+    // Current version is stable
+    if ($isStable) {
+        return [
+            'next_patch' => sprintf('%d.%d.%d', $major, $minor, $patch + 1),
+            'next_minor' => sprintf('%d.%d.0', $major, $minor + 1),
+            'next_major' => sprintf('%d.0.0', $major + 1),
+            'next_alpha' => sprintf('%d.0.0-alpha.1', $major + 1),
+            'next_beta' => sprintf('%d.0.0-beta.1', $major + 1),
```

Comment:
Can we update these to include the `v` prefix?

Reply:
This is [already the case](https://github.com/tempestphp/tempest-framework/pull/545/files#diff-ba4e74f7fdeafe09fab0dd8bd69b66578025911d8ea9403bf70182183fdd6600R115), or do you mean something else?

Reply:
Missed this. Nice!